### DISCUSSION TITLE
How to change config save location?

### DISCUSSION THREAD

**Discussion by ChrisTheSus** [jagrosh/MusicBot#1077] [Questions] [answered]
Hi. I am trying to run two of these at once in ubuntu. The only problem is that in ubuntu both of my music bots get/save their config to the home folder so when i try to use both at once i end up running the same bot twice (in windows it was fine, it got its config from the folder that the bot was located in). is there any way to change where the config is saved?

**Comment by MichailiK** [comment]
JMusicBot uses your working directory when accessing files. Use `cd` to change your working directory before running Java.

**Reply by ChrisTheSus** [reply]
Oh ok I'll try running it in the terminal. I was just right clicking it and opening as an executable with the java program I installed

**Comment by jagrosh** [comment, marked as answer]
Advanced config options are explained [here](https://jmusicbot.com/advanced-config/).

You can specify the location of your existing config location using the `-Dconfig=` flag, providing either an absolute or relative path. For example, `-Dconfig=configs/first_config.txt` would look for a config file called "first_config.txt" relative to your starting location in a folder called "configs". By default, JMusicBot looks for "config.txt" in the working directory.
